WireAnalytics/Sources/WireAnalytics/AnalyticsSession.swift

```diff
@@ -18,22 +18,82 @@
 
 import Countly
 
+/// A structure representing a user's profile for analytics purposes.
+
+public struct AnalyticsUserProfile {
+
+    /// The unique identifier for the user's analytics profile.
+    public let analyticsIdentifier: String
+
+    /// The identifier for the team the user belongs to.
+    public let teamID: String?
+
+    /// The role of the user within the team.
+    public let teamRole: String
+
+    /// The size of the team the user belongs to.
+    public let teamSize: Int?
```

Comment:
I'm guessing these three team related fields should all be nil together, or all not nil. If that's the case, you could group them into a new struct that is optional:

```swift
struct TeamInfo {

  let id: String
  let role: String
  let size: Int

}

let teamInfo: TeamInfo?
```

Reply:
Done: https://github.com/wireapp/wire-ios/commit/631633537b5cc28df055df80a54f050f0140969e